### ISSUE TITLE
UnicodeEncodeError on getTransactionListRequest

### ISSUE THREAD

**Issue by eechien** [AuthorizeNet/sdk-python#73]
I'm making a request to get a TransactionList from a batchId but am encountering a UnicodeEncodeError: 
`{UnicodeEncodeError}'ascii' codec can't encode characters in position 6631-6632: ordinal not in range(128)`
It's erroring on line 162, of apicontrollersbase.py:
`domResponse = xml.dom.minidom.parseString(self._httpResponse)`
self._httpResponse contains character u'\xe9'. So I was able to pass a unicode character to create a transaction but am not able to get that information back.

**Comment by eechien** [comment]
Hi, is someone looking into this issue? Thanks.

**Comment by eechien** [comment]
Hi @sunnyrajrathod, I saw that you put in a fix for this issue. Thank you! I just have one issue. I tried the new release and apicontractsv1.getTransactionListRequest()  is returning successfully for transactions that have a Transaction Status of "Settled Successfully" but when a transaction has a status of "Expired", then the call fails.

```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    auth_td.get_transactions(7098882)
  File "/Users/echien/code/website/nols_website/payment/authorize.py", line 142, in get_transactions
    response = self._send(self.RequestType.TRANSACTION_LIST, request)
  File "/Users/echien/code/website/nols_website/payment/authorize.py", line 114, in _send
    controller.execute()
  File "/Users/echien/code/virtualenvs/website/lib/python2.7/site-packages/authorizenet/apicontrollersbase.py", line 1
56, in execute
    domResponse = xml.dom.minidom.parseString(self._httpResponse)
  File "/usr/local/Cellar/python/2.7.12_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xml/dom/minidom.py",
line 1928, in parseString
    return expatbuilder.parseString(string)
  File "/usr/local/Cellar/python/2.7.12_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xml/dom/expatbuilder.
py", line 940, in parseString
    return builder.parseString(string)
  File "/usr/local/Cellar/python/2.7.12_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/xml/dom/expatbuilder.
py", line 223, in parseString
    parser.Parse(string, True)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 4187-4188: ordinal not in range(128)
```

**Comment by Sapbasu15** [comment]
@eechien , the issue is fixed in the latest release of the SDK. 
Please let us know if same unicode error is coming still.

**Comment by Sapbasu15** [comment]
Thank you for raising this issue. We have merged PR ( #78 ) and hope that should resolve unicode encoding issue. We are therefore closing this issue. Please raise a new issue, if you’re still facing problems, and we’ll look into it quickly.
